docs/UserGuide.md

```diff
@@ -152,6 +152,8 @@ Examples:
 
 - `find John` returns `john`, `John Doe` and `Johnny`
 - `find alex david` returns `Alex Yeoh`, `David Li`<br>
+
+**Result from `find alex david`:**
```

Comment:
I think there is a missing line break here.
![image](https://github.com/user-attachments/assets/4755c62f-a197-43f5-ba40-91f3ff3bfdcd)